bin/setup

```diff
@@ -2,7 +2,7 @@
 require "fileutils"
 
 # path to your application root.
-APP_ROOT = File.expand_path('..', __dir__)
+APP_ROOT = File.expand_path("..", __dir__)
```

Comment:
I think it would be easier to understand if the style change of `'` to `"` is made in a separate PR.

Reply:
@abetomo 
Thanks! I've just created the PR about only this changes at https://github.com/ranguba/ranguba/pull/10. Could you review it?